paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/CustomerSheetTest.kt

```diff
@@ -80,6 +82,90 @@ internal class CustomerSheetTest {
             method("POST"),
             path("/v1/payment_methods"),
             cardDetails(),
+            billingDetails(),
+        ) { response ->
+            response.testBodyFromFile("payment-methods-create.json")
+        }
+
+        networkRule.enqueue(
+            host("api.stripe.com"),
+            method("GET"),
+            path("/v1/setup_intents/seti_12345"),
+            query("client_secret", "seti_12345_secret_12345"),
+        ) { response ->
+            response.testBodyFromFile("setup-intent-get.json")
+        }
+
+        networkRule.enqueue(
+            host("api.stripe.com"),
+            method("POST"),
+            path("/v1/setup_intents/seti_12345/confirm"),
```

Comment:
I'm wondering if we should add helpers for these, similar to what we have for confirmParams. Maybe something like `confirmRequest`? Probably makes the most sense as a follow up.

Reply:
Yeah that makes sense! I'll add them in a follow-up.